backend/README.md

```diff
@@ -0,0 +1,17 @@
+# Backend
+
+## Data Ingest Simulator
+
+### Serial Port Emulator
+
+You can use any method to emulate a serial port that works for you, but the following steps worked for @Potato-Man114.
+
+1. Get the free 2-week trial of [Virual Serial Port Driver Pro](https://www.virtual-serial-port.org/).
```

Comment:
Is this supposed to be just ones for the list? Or is it possible to increment this a little more?

Reply:
Yes, this is supposed to be just `1.` for each item in the list. When the markdown is rendered, it will render with the correct numberrs. It removes the headache of numbering a list and changing it later.

![image](https://github.com/jak103/powerscore/assets/97547996/1d835655-11e4-402a-abd7-067245b3ebe6)